gnd/src/main/java/com/google/android/gnd/GndApplicationModule.java

```diff
@@ -45,8 +45,7 @@
 import javax.inject.Singleton;
 
 @Module(includes = {AndroidSupportInjectionModule.class, ViewModelModule.class})
-abstract class GndApplicationModule {
-  private static final String SHARED_PREFERENCES_NAME = "shared_prefs";
+public abstract class GndApplicationModule {
```

Comment:
Out of interest, what caused this to become `public`?

Reply:
At first, I reused the same variable `SHARED_PREFERENCES_NAME` in the SettingsFragment, but then moved it `Config.java`. So this can be reverted back now. Nice catch!